examples/shardingsphere-sample/shardingsphere-example-generator/src/main/java/org/apache/shardingsphere/example/generator/scenario/ExampleScenarioFactory.java

```diff
@@ -86,10 +90,39 @@ private FrameworkExampleScenario getFrameworkScenario(final String framework) {
     public Map<String, String> getResourceTemplateMap() {
         Map<String, String> result = new HashMap<>();
         for (FeatureExampleScenario each : featureScenarios) {
-            result.putAll(each.getJavaClassTemplateMap());
+            result.putAll(each.getResourceTemplateMap());
         }
         result.putAll(frameworkScenario.getResourceTemplateMap());
         result.put("resources/logback.ftl", "logback.xml");
         return result;
     }
+    
+    /**
+     * Get java class path collection.
+     *
+     * @return java class path collection.
+     */
+    public Collection<String> getJavaClassPaths() {
+        Collection<String> result = new HashSet<>();
+        for (FeatureExampleScenario each : featureScenarios) {
+            result.addAll(each.getJavaClassPaths());
+        }
+        result.addAll(frameworkScenario.getJavaClassPaths());
+        result.add("entity");
+        return result;
+    }
+    
+    /**
+     * Get resource path collection.
+     *
+     * @return resource path collection.
```

Comment:
1. Please change java from `path collection` to `paths`
2. @return should not add `.` in the end of line

Reply:
done.